api/v1alpha1/mustgather_types.go

```diff
@@ -53,12 +53,13 @@ type MustGatherSpec struct {
 	// +kubebuilder:validation:Optional
 	ProxyConfig ProxySpec `json:"proxyConfig,omitempty"`
 
-	// A time limit for gather command to complete a floating point number with an optional suffix:
-	// "s" for seconds (the default), "m" for minutes, "h" for hours or "d" for days will default to: $DEFAULT_MUST_GATHER_TIMEOUT'
+	// A time limit for gather command to complete a floating point number with a suffix:
+	// "s" for seconds, "m" for minutes, "h" for hours, or "d" for days.
+	// Will default to 0s
 	// +kubebuilder:validation:Optional
-	MustGatherTimeout string `json:"mustGatherTimeout,omitempty"`
+	MustGatherTimeout metav1.Duration `json:"mustGatherTimeout,omitempty"`
```

Comment:
No change needed, I think this is the right approach, but we may want to review any SOPs/ensure MCS is aware that the API is changing in case someone has been setting the timeout field.